src/utils/get-core-member-logins.ts

```diff
@@ -29,7 +29,7 @@ export const getCoreMemberLogins = async (pull_number: number, teams?: string[])
 export const getRequiredCodeOwnersEntries = async (pull_number: number): Promise<CodeOwnersEntry[]> => {
   const codeOwners = (await loadOwners(process.cwd())) ?? [];
   const changedFilePaths = await getChangedFilepaths(pull_number);
-  return changedFilePaths.map(filePath => matchFile(filePath, codeOwners)).filter(Boolean);
+  return changedFilePaths.map(filePath => matchFile(filePath, codeOwners)).filter(Boolean) as CodeOwnersEntry[];
```

Comment:
You removed so many casts, but added this one! 😂 What for?

Reply:
typescript isn't understanding that the result of this is actually a `CodeOwnersEntry[]` and not a `(CodeOwnersEntry | null)[]`. i hate typecasting but i also didn't want to rewrite this very readable line to placate ts